Packs/GmailSingleUser/pack_metadata.json

```diff
@@ -18,6 +18,11 @@
     "deprecated": false,
     "useCases": [],
     "keywords": [],
-    "dependencies": {},
+    "dependencies": {
```

Comment:
Will this cause a circular dependency ?

Reply:
@yuvalbenshalom ?

Reply:
@aviadl this may create circular dependency, but as I understood from @asafshen the server should be able to handle it.

Reply:
yep, we handle circular dependencies 

fyi, circular dependencies make sense in some manner
you can have:
 - pack-A with
    - Playbook-A that uses Script-B
    - Script-A
 - pack-B with
    - Playbook-B that uses Script-A
    - Script-B


there are also longer chains of circular dependencies that makes more sense